common/RefereeEnums.h

```diff
@@ -0,0 +1,96 @@
+#ifndef GT_ROBOJACKETS_ROBOCUP_REFEREEENUMS_H
+#define GT_ROBOJACKETS_ROBOCUP_REFEREEENUMS_H
```

Comment:
I think `#pragma once` and `RefereeEnums.hpp` match the rest of the code base better

Reply:
That's true.